src/base.jl

```diff
@@ -64,7 +66,10 @@ function System(buses::Vector{Bus},
                 loads::Vector{<:ElectricLoad},
                 branches::Union{Nothing, Vector{<:Branch}},
                 storage::Union{Nothing, Vector{<:Storage}},
-                basepower::Float64; kwargs...)
+                basepower::Float64,
+                forecasts::Union{Nothing, SystemForecasts},
+                services::Union{Nothing, Vector{ <: Service}},
+                annex::Union{Nothing,Dict{Any,Any}}; kwargs...)
     runchecks = in(:runchecks, keys(kwargs)) ? kwargs[:runchecks] : true
```

Comment:
Prefer this idiom: `get(kwargs, :runchecks, true)` instead?

Reply:
I vote that we defer conversation about refactoring System constructors.  Once we integrate workflows with ConcreteSystem in PowerSimulations and other consumers, we need to look at changing how users construct ConcreteSystem.  They should not have to construct System first.  In summary, ConcreteSystem will ultimately become System and will need new constructors.